source/slang/slang.vcxproj

```diff
@@ -1,415 +1,417 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should double-check why this diff is so massive.

In general, the Right Way to generate the project files is to invoke the copy of `premake5.exe` that is checked into the `slang-binaries` submodule. By doing this, we can guarantee that all the developers/contributors are on a consistent version of the tool, which we hope will generate project files with minimal diffs.